tastypie/test.py

```diff
@@ -572,3 +573,18 @@ def assertKeys(self, data, expected):
         changes.
         """
         self.assertEqual(sorted(data.keys()), sorted(expected))
+
+
+
+class ResourceTestCase(ResourceTestCaseMixin, TestCase):
+    """
+    Subclass Django's TestCase class with useful methods for testing Tastypie APIs.
+    """
+    pass
+
+
+class ResourceTransactionTestCase(ResourceTestCaseMixin, TransactionTestCase):
+    """
+    Subclass Django's TransactionTestCase class with useful methods for testing Tastypie APIs.
+    """
+    pass
```

Comment:
`./tastypie/test.py:590:9: W292 no newline at end of file` [#2](https://travis-ci.org/django-tastypie/django-tastypie/jobs/97561980)